github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-postsubmits.yaml

```diff
@@ -65,3 +65,145 @@ postsubmits:
           resources:
             requests:
               memory: "29Gi"
+    - name: publish-fedora-with-test-tooling
+      branches:
+      - main
+      always_run: false
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      decoration_config:
+        timeout: 2h
+      run_if_changed: cluster-provision/images/vm-image-builder/.*\.sh$ | cluster-provision/images/vm-image-builder/fedora-with-test-tooling/.*
+      max_concurrency: 1
+      extra_refs:
+      - org: kubevirt
+        repo: project-infra
+        base_ref: main
+      labels:
+        preset-dind-enabled: "true"
```

Comment:
This preset is not required - see `preset-podman-in-container-enabled: "true"` below

Reply:
Ok, so preset-dind-enabled means enable Docker in Docker,
```
- labels:
    preset-dind-enabled: "true"
  env:
  - name: DOCKER_IN_DOCKER_ENABLED
    value: "true"
  volumes:
  - name: docker-graph
    emptyDir: {}
  volumeMounts:
  - name: docker-graph
    mountPath: /docker-graph
```